code/modules/integrated_electronics/core/helpers.dm

```diff
@@ -69,4 +69,56 @@
 		if(pin)
 			debugger.write_data(pin, usr)
 			return 1
-	return 0
\ No newline at end of file
+	return 0
+
+/obj/item/integrated_circuit/proc/asc2b64(var/S)
```

Comment:
What do these circuits do?

Reply:
It's no circuits. It's helper functions,which converts string to base 64 and reverse. I was planning to implement it further.

Reply:
We already have a string-ascii converter, which allows for more characters?

Reply:
string to base 64. https://en.wikipedia.org/wiki/Base64